CMakeLists.txt

```diff
@@ -727,6 +736,22 @@ elseif("${SWIFT_HOST_VARIANT_SDK}" STREQUAL "HAIKU")
   set(SWIFT_PRIMARY_VARIANT_SDK_default  "${SWIFT_HOST_VARIANT_SDK}")
   set(SWIFT_PRIMARY_VARIANT_ARCH_default "${SWIFT_HOST_VARIANT_ARCH}")
 
+elseif("${SWIFT_HOST_VARIANT_SDK}" STREQUAL "ANDROID")
+
+  set(SWIFT_HOST_VARIANT "android" CACHE STRING
+      "Deployment OS for Swift host tools (the compiler) [android]")
+
+  set(ANDROID_NATIVE_SYSROOT "/data/data/com.termux/files" CACHE STRING
```

Comment:
Please don't set this variable, we want this to be specified by the user since there is no standard location for it.

Reply:
I believe it can still be overridden by the user, if passed in by CMake? Given that all subsequent uses of this variable in this pull are keyed to this Termux path and it's likely the only popular build environment on Android, this seems a reasonable default.

Reply:
Yes, it can be overridden by making it a cached variable

Reply:
It's already a cached variable now, so I guess nothing needs to be changed.